destination/src/test/scala/quasar/plugin/sqlserver/destination/SQLServerDestinationSeekSinksSpec.scala

```diff
@@ -82,7 +82,7 @@ object SQLServerDestinationSeekSinksSpec extends EffectfulQSpec[IO] with BeforeA
   val Mod = SQLServerDestinationModule
 
   "seek sinks (upsert and append)" should {
-    val XStringCol = Column("x", SQLServerType.NVARCHAR(255))
+    val XStringCol = Column("x", SQLServerType.TEXT)
```

Comment:
Is this change here to show that we can work with `TEXT` ids? Maybe provide a test checking that it's `CHAR(900)` after pushing?

Reply:
Yes, `TEXT` appears to be generally unindexable (at least with the out-of-the-box settings), without the changes in this PR, all the tests that create indexes will fail. You're right that this is a bit implicit though, I'll add an explicit test for the adjusted column type.

Reply:
Thanks!